data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -16,13 +16,15 @@ const NotFound = errors.NotFound;
 const Conflict = errors.Conflict;
 const InternalServerError = errors.InternalServerError;
 
-const apiserver = new kc.Client({
-  config: {
+const apiserverConfig = config.apiserver.pathToKubeConfig ?
```

Comment:
check how the kubeconfig can be found from within the pod itself.

Can you try this? https://github.com/godaddy/kubernetes-client/blob/d17bb81cdf4c45289b10aba84c586f07bfdf0b75/test/config.test.js#L44

Reply:
let's stick to the current approach for now.